devel/api/integration/index.rst

```diff
@@ -533,3 +533,54 @@ The ``status_url`` property returns the URL to track kthe progress of the reques
 
 The operation will be completed once the ``status`` property is updated with the value ``finished``.
 
+Dataset Get Metadata 
```

Comment:
rename to 'Dataset Get standardized Metadata'

and follow @giohappy s comment https://github.com/GeoNode/documentation/pull/264#issuecomment-1316817796

> An example of how to retrieve the actual standard metadata could be added, which can be obtained from the `links` array inside the detail response.

Retrieving the Ressource Details is already described here (Example 2.)
https://docs.geonode.org/en/master/devel/api/integration/index.html#example-requests